src/lua/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+
+list(APPEND LUA_SRC_FOLDERS
+	${CMAKE_CURRENT_SOURCE_DIR}
+	core
+)
+
+# Creates variables LUA_CXX_FILES and LUA_HXX_FILES
+add_source_folders(LUA LUA_SRC_FOLDERS)
+
+# Creates a library, adds it to the build, and sets C++ target properties on it
+define_pioneer_library(pioneer-lua LUA_CXX_FILES LUA_HXX_FILES)
+target_include_directories(pioneer-lua PRIVATE ${CMAKE_BINARY_DIR}/src)
```

Comment:
You don't include ```${CMAKE_BINARY_DIR}```.

Reply:
As in I shouldn't include that director, or I'm not doing it right now? Testing shows a copy of `buildopts.h` is being generated at several levels in `build/`, and I'm not 100% sure why it's being created in those different places...